f = open("None", "r")

print("예외처리 성공")

#try:
#    오류가 발생할 수 있는 구문
#except Exception as e:
#    오류발생
#else:
#    오류가 발생하지 않았을 때 출력하는 구문
#finally:
#    무조건 맨 마지막에 실행되는 구문